docs/openapi.yaml

```diff
@@ -260,6 +261,403 @@ paths:
         "504":
           $ref: "#/components/responses/TimeoutError"
 
+  "/v3/agreements":
+    get:
+      tags:
+        - Agreement V3 Endpoints
+      summary: List Agreements
+      operationId: ListAgreementsV3
+      description: >-
+        The API call allows merchant to fetch all agreements.
+        If no query status is supplied it will default to only retrieving active agreements.
+        There is no way to list all Agreements with all statuses, this is due to performance.
+      parameters:
+        - $ref: "#/components/parameters/Authorization"
+        - $ref: "#/components/parameters/Ocp-Apim-Subscription-Key"
+        - $ref: "#/components/parameters/Content-Type"
+        - $ref: "#/components/parameters/Merchant-Serial-Number"
+        - $ref: "#/components/parameters/Vipps-System-Name"
+        - $ref: "#/components/parameters/Vipps-System-Version"
+        - $ref: "#/components/parameters/Vipps-System-Plugin-Name"
+        - $ref: "#/components/parameters/Vipps-System-Plugin-Version"
+        - name: status
+          in: query
+          required: false
+          description: Filter by status of the agreement.
+          schema:
+            $ref: "#/components/schemas/AgreementStatus"
+        - name: createdAfter
+          in: query
+          description: Filter by agreement start date in miliseconds.
+          required: false
+          allowEmptyValue: false
+          schema:
+            type: integer
+          example: 1644572442944
+        - name: filter
+          in: query
+          description: Filter on agreement. Currently only supporting 'ACTIVE_CAMPAIGNS' filter
+          required: false
+          allowEmptyValue: false
+          schema:
+            type: string
+          example: ACTIVE_CAMPAIGNS
```

Comment:
remove since we currently does not support this (an I would argue we should not support filtering in this manner, but rather in a more semantic manner)

Reply:
Good catch, removed